subsys/config/src/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+zephyr_include_directories(${ZEPHYR_BASE}/ext/utils)
+
+#mbedtls-base64 library header
+zephyr_include_directories($ENV{ZEPHYR_BASE}/ext/lib/crypto/mbedtls/include)
+
+zephyr_include_directories($ENV{ZEPHYR_BASE}/ext/fs/nffs/include)
```

Comment:
dependency on two external module header files :(
You can remove this dependency as it will for work for NFFS and will get an error for FATFS from VFS.

Reply:
without this I've got build error:
`
In file included from /home/anpu/codedev/zephyr_3/include/fs.h:12:0,
                 from /home/anpu/codedev/zephyr_3/samples/hello_world/src/main.c:11:
/home/anpu/codedev/zephyr_3/include/fs/fs_interface.h:15:23: fatal error: nffs/nffs.h: No such file or directory
 #include <nffs/nffs.h>
`
I can fix it by adding $ENV{ZEPHYR_BASE}/ext/fs/nffs/include to include path for the test project.
Can you comment on this? - It looks like I'm doing this not as should it be  done.

Reply:
you need to select the NFFS and related flash device in test application. At subsystem level, it should only need fs.h where it refers to fs_* API's.